nx/lib/nx/defn/kernel.ex

```diff
@@ -411,6 +411,21 @@ defmodule Nx.Defn.Kernel do
   def left * right when Kernel.and(is_number(left), is_number(right)), do: Kernel.*(left, right)
   def left * right, do: Nx.multiply(left, right)
 
+  @doc """
+  Element-wise multiplication operator.
+
+  It delegates to `Nx.power/2` (supports broadcasting).
+
+  ## Examples
+
+      defn power(a, b) do
+        a ** b
+      end
+
+  """
+  def left ** right when Kernel.and(is_number(left), is_number(right)), do: Kernel.**(left, right)
```

Comment:
```suggestion
  def unquote(:"**")(left, right) when Kernel.and(is_number(left), is_number(right)), do: Kernel.**(left, right)
```

Please change all occurences to something like this so it compiles on Elixir 1.12

Reply:
I think we can bump to 1.13, especially because it will allow us to use the `~M` and `~V` sigils!

Reply:
Sure, makes sense!